src/atoms/ApmGallery/ApmGallery.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, cleanup, prettyDOM } from '@testing-library/react';
+import Body from '../../components/Body/Body';
+import gallery from '../../fixtures/gallery.json';
```

Comment:
maybe the import is failing? try: import * as gallery from '../../fixtures/gallery.json';

Reply:
I tried that, but that doesn't seem to be the issue. It says the Traverse function is failing here, even though it appears to work just fine in all other tests and render fine when used on the News site. I'm guessing my gallery test data is slightly wrong, even though I tried to ensure that it was just the right shape. I spent too much time on it and decided to save it for later for the time being.